acceptance/suites/pre_suite/puppet3_compat/71_install_simmons_module.rb

```diff
@@ -1,4 +1,4 @@
-simmons_version = '0.3.1'
+simmons_version = '0.3.2'
```

Comment:
Dangit, I screwed up some stuff with the Forge and module release - @camlow325 could you bump this to `0.3.3` please?